src/commonMain/kotlin/KMPJsonLogicEngine.kt

```diff
@@ -0,0 +1,3 @@
+object KMPJsonLogicEngine {
+    fun evaluate(expression: String, data: Map<String, String>) = true
```

Comment:
trzeba bedzie jeszcez dogadac jaki typ bedzie szedł jako expression (rozmawialismy o tym na dejli)

Reply:
1) I think it's better to define some interface public interface and implement it in `KMPJsonLogicEngine`.
2) Let's try to replace expression type `String` with `Map<String, List<Any>>`, as we decided at the meeting.
3) How about returning/throwing the error when the evaluation failed?
4) Do we need KMP prefix? 🤔

Reply:
1. 👍 
2. 👍 but it probably will be `Map<String, Any>` since there are operators which have only one parameter and does not require passing them in array e.g. `{"var":"a"}` to retrieve data from registry or `{"!": true}`
3. 👍 - and I guess this error would be catched in MBox libraries, right?
4. 👍 I think we don't need it

The other thing is that this function probably will also return Any, not Boolean (or maybe String?)

Reply:
@PaulinaSadowskaAllegro  
2. You're right 
3. Yes

Maybe it's better to return `String`, because in `data` we also have `String` values

Reply:
I did the basics, wdyt about introducing these changes by people who are working on it at the moment? Lets keep the scope narrow and discuss in next PRs

Reply:
1. we should put everything we can that will be 'usable' outside as interface 👍 
2. Map<String, Any> should be fine too 👍 
3. we probably will need to make own error type. Not sure how ios handles those (checked vs uncheched in java and kotlin)